tests/factories/brac/contacts/brac-place.js

```diff
@@ -0,0 +1,45 @@
+const Factory = require('rosie').Factory;
+const uuid = require('uuid');
+const Faker = require('@faker-js/faker');
+const householdSurveyFactory = require('../reports/create-family-household-survey');
+
+const bracPlace = () => {
+  return new Factory()
+    .sequence('_id', uuid.v4)
+    .attr('parent', '')
+    .attr('type', '')
+    .attr('is_name_generated', 'true')
+    .attr('name', '')
```

Comment:
I thought it would be a good idea if the process defines the place name based on the thread name to ensure that the place name is always unique (line 13 on generate-brac-data.js). What do you think?

Reply:
That sounds good :)